packages/block-library/src/navigation/index.php

```diff
@@ -5,148 +5,6 @@
  * @package WordPress
  */
 
-// These functions are used for the __unstableLocation feature and only active
```

Comment:
I think the failing E2E's are related 😢 

So they could also either be gated, or even more interesting, a filter could take over the role: https://github.com/WordPress/gutenberg/pull/52579